SS_write_ssnew.tpl

```diff
@@ -1223,19 +1255,25 @@ FUNCTION void write_nudata()
         report1 << "-9999 0 0" << endl;
       }
 
-      report1 << "#" << endl
-              << SzFreq_Nmeth << " # N sizefreq methods to read " << endl;
-      if (SzFreq_Nmeth > 0)
-      {
-        report1 << SzFreq_Nbins << " #Sizefreq N bins per method" << endl;
-        report1 << SzFreq_units << " #Sizetfreq units(1=bio/2=num) per method" << endl;
-        report1 << SzFreq_scale << " #Sizefreq scale(1=kg/2=lbs/3=cm/4=inches) per method" << endl;
-        report1 << SzFreq_mincomp << " #Sizefreq:  add small constant to comps, per method " << endl;
+      report1 << "#" << endl << "# Sizefreq data. Defined by method because a fleet can use multiple methods" << endl;
+      report1 << SzFreq_Nmeth_rd << " # N sizefreq methods to read (or -1 for expanded options)" << endl;
+      if (SzFreq_Nmeth_rd < 0) {
+        report1 << SzFreq_Nmeth << " # N sizefreq methods to read" << endl;
+      }
+      if (SzFreq_Nmeth != 0) {
+        report1 << "# each row below has entry for each sizefreq method " << endl;
+        report1 << SzFreq_Nbins << " #Sizefreq N bins" << endl;
+        report1 << SzFreq_units << " #Sizetfreq units(1=bio/2=num)" << endl;
+        report1 << SzFreq_scale << " #Sizefreq scale(1=kg/2=lbs/3=cm/4=inches)" << endl;
+        report1 << SzFreq_mincomp << " #Sizefreq:  small constant to add to comps" << endl;
         report1 << SzFreq_nobs << " #Sizefreq N obs per method" << endl;
+        if (SzFreq_Nmeth_rd < 0) {
+          report1 << Comp_Err_Sz <<  " #_Comp_Error:  0=multinomial, 1=dirichlet using Theta*n, 2=dirichlet using beta, 3=MV_Tweedie" << endl;
+          report1 << Comp_Err_Sz2 << " #_Comp_Error: index for dirichlet or MV_Tweedie" << endl;
```

Comment:
Can we change the comment from
"#_Comp_Error: index for dirichlet or MV_Tweedie"
to 
"#_ParmSelect:  consecutive index for dirichlet or MV_Tweedie"
to avoid confusion among these two new inputs lines and better match the comments for other data types?

Reply:
done, plus some additional related changes to those sections